tools/flaky-test-reporter/ghutil/fakeghutil/fakeghutil.go

```diff
@@ -70,12 +80,20 @@ func (fgc *FakeGithubClient) ListIssuesByRepo(org, repo string, labels []string)
 // CreateIssue creates issue
 func (fgc *FakeGithubClient) CreateIssue(org, repo, title, body string) (*github.Issue, error) {
 	issueNumber := fgc.getNextNumber()
+	stateStr := string(ghutil.IssueOpenState)
+	repoURL := fmt.Sprintf("%s/%s/%s", fgc.BaseURL, org, repo)
+	url := fmt.Sprintf("%s/%d", repoURL, issueNumber)
 	newIssue := &github.Issue{
-		Title:  &title,
-		Body:   &body,
-		Number: &issueNumber,
+		Title:         &title,
+		Body:          &body,
+		Number:        &issueNumber,
+		State:         &stateStr,
+		URL:           &url,
```

Comment:
Nit: If you are "url" is not used anywhere further in the code, avoid creating the variable url and initialize it here.

Reply:
this is a field of *github.Issue, so can't rename it